app/routes/reductions.js

```diff
@@ -159,46 +160,49 @@ module.exports = function (router) {
 
     var id = req.params.id
     var reduction
-    try {
-      reduction = generateNewReductionFromRequest(req.body)
-    } catch (error) {
-      if (error instanceof ValidationError) {
-        return reductionsService.getAddReductionsRefData(id, organisationLevel)
-          .then(function (result) {
-            return res.status(400).render('add-reduction', {
-              breadcrumbs: result.breadcrumbs,
-              linkId: id,
-              title: result.title,
-              subTitle: result.subTitle,
-              subNav: getSubNav(id, organisationLevel, req.path),
-              referenceData: result.referenceData,
-              reduction: {
-                id: req.body.reductionId,
-                reasonId: req.body.reasonForReductionId,
-                hours: req.body.reductionHours,
-                start_day: req.body.redStartDay,
-                start_month: req.body.redStartMonth,
-                start_year: req.body.redStartYear,
-                end_day: req.body.redEndDay,
-                end_month: req.body.redEndMonth,
-                end_year: req.body.redEndYear,
-                notes: req.body.notes
-              },
-              errors: error.validationErrors
-            })
-          }).catch(function (error) {
-            next(error)
+    var reductionReason
+
+    return reductionsService.getAddReductionsRefData(id, organisationLevel)
```

Comment:
Could you write a service to return just that reduction reason?  Rather than return them all, plus title/subtitle, plus going to look up workload_owners contracted hours etc

Reply:
The `result` is used in the `catch` below for passing back to the view. I just moved the function up a few lines to steal its retuned value